.github/workflows/generators.yml

```diff
@@ -0,0 +1,91 @@
+name: generators
+
+on: workflow_dispatch
+
+jobs:
+  generators:
+    name: Test generators
+    runs-on: ubuntu-latest
+
+    container:
+      image: ghcr.io/n3pdf/lhapdf:v2
+      credentials:
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # tags needed for dynamic versioning
+          fetch-depth: 0
+      - name: Install and configure Poetry
+        uses: snok/install-poetry@v1
+        with:
+          virtualenvs-create: false
+          installer-parallel: true
+      - name: Install project
+        run: |
+          poetry install --no-interaction
+      - name: Get pinecards
+        run: |
+          cd examples && git clone git@github.com:NNPDF/pinecards.git
+      - name: Get a theory
+        uses: DamianReeves/write-file-action@master
+        with:
+          path: examples/401.yaml
+          write-mode: overwrite
```

Comment:
Instead of this, put a file in a `test/` folder

Reply:
well, it is not a unit test, so I need a better name, but yeah ...

Reply:
The rationale is that:
- things should be easily reproducible also off-line, otherwise debugging is a nightmare, and
- you're mixing data and code, and this clutters the workflow file